OIPA/common/download_file.py

```diff
@@ -0,0 +1,300 @@
+"""Downloads files from http or ftp locations.
```

Comment:
@HatimKH you could also create an issue on the projects GitHub if there's an error in 0.4.0; https://github.com/bantonj/fileDownloader